web_src/js/features/captcha.js

```diff
@@ -0,0 +1,51 @@
+import {isDarkTheme} from '../utils.js';
+
+export async function initCaptcha() {
+  const captchaEl = document.querySelector('#captcha');
+  if (!captchaEl) return;
+
+  const siteKey = captchaEl.getAttribute('data-sitekey');
+  const isDark = isDarkTheme();
+
+  const params = {
+    sitekey: siteKey,
+    theme: isDark ? 'dark' : 'light'
+  };
+
+  switch (captchaEl.getAttribute('captcha-type')) {
```

Comment:
Non-standard attributes usually need the `data-` prefix.

Reply:
It was fixed in this commit https://github.com/go-gitea/gitea/pull/22369/commits/4b3b828c8d308a558b0fe5cfb35c8276c9e13214